src/nurse/models.py

```diff
@@ -74,3 +74,16 @@ def __str__(self):
     def is_valid(self):
         """Returns true if the prescription is still valid (e.g. hasn't expired)."""
         return self.start_date <= datetime.now().date() <= self.end_date
+
+
+class UserOneSignalProfileManager(models.Manager):
+    def subscription_ids_for_users(self, users):
+        return list(
+            self.filter(user__in=users).values_list("subscription_id", flat=True)
+        )
```

Comment:
I think we should keep it a QuerySet and don't force the evaluation within the Manager.
i.e. remove the `list()`

Reply:
return the QuerySet without converting it to a list